plugins/controlboard/src/ControlBoard.cc

```diff
@@ -110,15 +110,15 @@ GZ_REGISTER_MODEL_PLUGIN(GazeboYarpControlBoard)
         }
         #ifndef GAZEBO_YARP_PLUGINS_DISABLE_IMPLICIT_NETWORK_WRAPPERS
         bool disable_wrapper = m_parameters.check("disableImplicitNetworkWrapper");
-        
+
         if (disable_wrapper && !m_parameters.check("yarpDeviceName"))
         {
             yError() << "GazeboYarpControlBoard : missing yarpDeviceName parameter for one device in robot " << m_robotName;
             return;
         }
-        
```

Comment:
For the future, please avoid functional and whitespace changes in the same commit, thanks!